### ISSUE TITLE
Stream Queue

### ISSUE THREAD

**Issue by toasty-spl** [Sendouc/sendou.ink#1871]
Marking multiple rounds to be on stream during Round Robin stages

**Comment by Sendouc** [comment]
Yes I think this would be good. If people want to show screenshots and explain how it works with start.gg (in case it's a good implementation) then that might be nice for reference. Not sure if Battlefy has it.

**Comment by toasty-spl** [comment]
I had this screenshot as well in DMs with Sendou for reference on what the buttons end up doing 

![image](https://github.com/user-attachments/assets/5c9f06b1-a763-41ac-9d71-29ef61c3d865)

This way the order can be adjusted from the TO/Admin side as needed if sets end up shuffling around for some reason.

**Comment by im-xi** [comment]
For the API, it'd be nice to have the upcoming maps where applicable (e.g. for prepared maps like [this](https://github.com/user-attachments/assets/2fd0b694-ea22-4317-9302-987455db26b5), as currently it returns null for any match that has yet to start.
